src/search/FindInFiles.js

```diff
@@ -93,7 +93,19 @@ define(function (require, exports, module) {
     }
 
     function nodeFileCacheComplete(event, numFiles, cacheSize) {
-        var projectName = ProjectManager.getProjectRoot().name || "noName00";
+        if (/\/test\/SpecRunner\.html$/.test(window.location.pathname)) {
```

Comment:
Would be nice to check for the `/brackets/i` regex too. But I don't know the location on the other OS.